tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot

```diff
@@ -0,0 +1,346 @@
+*** Settings ***
+Library             ../../libs/admin_api.py
+Resource            ../../libs/admin-common.robot
+Resource            ../../libs/public-common.robot
+
+Suite Setup         user signs in as bau1
+Suite Teardown      user closes the browser
+Test Setup          fail test fast if required
+
+Force Tags          Admin    Local    Dev    AltersData
+
+*** Variables ***
+${PUBLICATION_NAME_ARCHIVE}=        UI tests - archived publication %{RUN_IDENTIFIER}
+${RELEASE_NAME_ARCHIVE}=            Financial Year 3000-01
+${SUBJECT_NAME_ARCHIVE}=            Subject for archived publication
+
+${PUBLICATION_NAME_SUPERSEDE}=      UI tests - superseding publication %{RUN_IDENTIFIER}
+${RELEASE_NAME_SUPERSEDE}=          Financial Year 2000-01
+${SUBJECT_NAME_SUPERSEDE}=          Subject for superseding publication
+
+*** Test Cases ***
+Create new publication to be archived and release via API
+    ${PUB_ID_ARCHIVE}=    user creates test publication via api    ${PUBLICATION_NAME_ARCHIVE}
```

Comment:
I would change this to `PUBLICATION_ID_ARCHIVE` to make it more obvious that this means publication and not public